components/ui/AppBasicCard.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-  <article class="learning-resource-card">
-    <img
-      class="learning-resource-card__image"
-      alt=""
-      src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
+  <article class="app-card">
+    <div
+      class="app-card__image"
```

Comment:
The image is the background of a `div` not of an `img`, and `div` tags don't have `alt` attribute, that's why I remove it. Anyways, we could try one of [these solutions](https://stackoverflow.com/questions/38380115/is-there-an-equivalent-to-the-alt-attribute-for-div-elements), what do you think?

Reply:
You're right, this is a `div` and not an `img`... 

I take that back now, I think since these are purely _decorative_ images, and since it's not also an image element, we don't need that here.